jupyter/tensorflow/ubi9-python-3.11/Pipfile

```diff
@@ -23,7 +23,8 @@ scikit-learn = "~=1.5.1"
 scipy = "~=1.14.1"
 skl2onnx = "~=1.17.0"
 onnxconverter-common = "~=1.13.0" # Required for skl2onnx, as upgraded version is not compatible with protobuf
```

Comment:
shouldn't this be removed/upgraded?

Reply:
The onnxconverter-common can't be upgraded as it has also pinned protobuf in its latest version 
https://github.com/microsoft/onnxconverter-common/blob/8d2f85f0a7039a0b0defd6c7509cb905914a4d2f/pyproject.toml#L27
if we do want to fix this, it needs to be fixed in all images
so do think it fits to this PR.

Reply:
if it's still relevant, then I have no objections

Reply:
yes, still relevant